internal/compose/compose.go

```diff
@@ -192,14 +196,19 @@ func NewProject(name string, paths ...string) (*Project, error) {
 		c.disableANSI = true
 	}
 
+	v, ok = os.LookupEnv(DisablePullProgressInformationEnv)
+	if ok && strings.ToLower(v) != "false" {
+		c.disablePullProgressInformation = true
+	}
+
 	return &c, nil
 }
 
 // Up brings up a Docker Compose project.
 func (p *Project) Up(opts CommandOptions) error {
 	args := p.baseArgs()
 	args = append(args, "up")
-	if p.disableANSI {
+	if p.disablePullProgressInformation {
 		args = append(args, "--quiet-pull")
```

Comment:
Are these flags available in compose V1? ([`disableANSI` is not used in V1](https://github.com/elastic/elastic-package/pull/1330/files#diff-8019656c843609e3cb15ca57ab96dc41fdadae8215d0ff22d9b989f795aef4a4R195))

Reply:
I've tested with two different versions of docker-compose v1 and they were present those versions.

Depending on the command the parameter changes `--quiet` or `--quiet`:

```bash
 $ ./docker-compose-1.25.5 --version
docker-compose version 1.25.5, build 8a1c60f6
 $ ./docker-compose-1.25.5 up -h  | grep quiet
    --quiet-pull               Pull without printing progress information
 $ ./docker-compose-1.25.5 pull -h  | grep quiet
    -q, --quiet             Pull without printing progress information

 $ ./docker-compose-1.29.2 --version
docker-compose version 1.29.2, build 5becea4c
 $ ./docker-compose-1.29.2 pull -h  | grep quiet
    -q, --quiet             Pull without printing progress information
 $ ./docker-compose-1.29.2 up -h  | grep quiet
    --quiet-pull               Pull without printing progress information
``` 

And these parameters are the same when running docker-compose v2:
```bash
 $ ./docker-compose-2.17.2 --version
Docker Compose version v2.17.2
 $ ./docker-compose-2.17.2 pull -h  | grep quiet
  -q, --quiet                  Pull without printing progress information.
 $ ./docker-compose-2.17.2 up -h  | grep quiet
      --quiet-pull                Pull without printing progress information.
```